### PR TITLE
Ana/add create account in testnets

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

All the intellectual property rights go to @jbibla 😜

OK, you probably want my solution here too much either because I used _that_ "funky" string destructuring to tell if a network is a testnet or not.

But I mean, to do this check in this component I would need to call Apollo and this is much cheaper computaniolly speaking...

Another option would be to pass this info (testnet or not) as a param through routing.

Do you have any other better solution?

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/components/common/TmSessionSignUp.vue [hunk 1]
```diff
@@ -4,7 +4,14 @@
       <h2 class="session-title">
         Create a new address
       </h2>
-      <div v-if="!session.insecureMode && !session.mobile" class="session-main">
+      <div
+        v-if="
+          !session.insecureMode &&
+            !session.mobile &&
+            !(network.slice(-7) === `testnet`)
```
Comment (jbibla) [inline src/components/common/TmSessionSignUp.vue]: 🤢

Comment (jbibla) [reply src/components/common/TmSessionSignUp.vue]: i think @iambeone made a helper function to check if it's a network - maybe we can use that or something?

Comment (faboweb) [reply src/components/common/TmSessionSignUp.vue]: we have the `testnet` property for checking if this is a testnet

Comment (Bitcoinera) [reply src/components/common/TmSessionSignUp.vue]: > 🤢

xD a bit exaggerated no?

File: src/components/common/TmSessionSignUp.vue [hunk 2]
```diff
@@ -107,6 +107,10 @@ export default {
       set(value) {
         this.$store.commit(`updateField`, { field: `signUpName`, value })
       }
+    },
+    isTestnet() {
+      // this.networks is undefined here O.o why??
+      return this.networks.filter(({ id }) => id === this.network)[0].testnet
```
Comment (faboweb) [inline src/components/common/TmSessionSignUp.vue]:
```suggestion
      return this.networks.find(({ id }) => id === this.network).testnet
```

Comment (faboweb) [reply src/components/common/TmSessionSignUp.vue]: don't really know why this.networks is undefined tough

Comment (Bitcoinera) [reply src/components/common/TmSessionSignUp.vue]: yes, I know. But your solution works. I don't understand it either 🤷‍♀

**Comment by Bitcoinera** [comment]
Codecov is not working?

File: src/components/common/TmSessionSignUp.vue (hunk 1 above)
Comment (jbibla) [reply src/components/common/TmSessionSignUp.vue]: yes, very exaggerated.

**Comment by jbibla** [comment]
might be waiting for the changelog 🙃

**Comment by Bitcoinera** [comment]
> might be waiting for the changelog 🙃

xD

**Pull request merged**